canonical/lxd-ui
# feat: semiotics update overview pages [WD-15624]

## Done

- Updated all resource detail overview pages with new icons i.e. instance overview, profile overview, network overview, storage pool overview, custom volume overview
- Adjusted instance overview devices section to hide `none` devices and show non-root disk devices.

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Go through instance overview, profile overview, network overview and storage pool overview and custom volume overview pages and make sure everything is working as expected.
